docs/PRODUCTION.md

```diff
@@ -13,3 +13,11 @@ The postgres and helm charts are included as conditionals in the Gen3 [umbrella
   repository: "https://charts.bitnami.com/bitnami"
   condition: global.dev
   ```
+
+### Kubernetes Configmap and Secret Configuration
+For the seamless operation of our services, we utilize Kubernetes secrets. To streamline the integration and management of these secrets, we highly recommend deploying External Secret Manager alongside any existing secret management systems you may already have in place. For a comprehensive guide and best practices on implementing External Secrets within our ecosystem, please consult our dedicated External Secrets Documentation available [here](https://github.com/uc-cdis/gen3-helm/blob/feat/GPE-1032/docs/external_secrets.md).
```

Comment:
reminder to update this link for when this gets merged so it's not pointing at the branch